lib/visitor-keys.js

```diff
@@ -7,36 +24,27 @@ const KEYS = {
         "left",
         "right"
     ],
-    ArrayExpression: [
-        "elements"
-    ],
-    ArrayPattern: [
-        "elements"
-    ],
-    ArrowFunctionExpression: [
-        "params",
-        "body"
+    AssignmentProperty: [
+        "key",
+        "value"
     ],
     AwaitExpression: [
         "argument"
     ],
-    BlockStatement: [
-        "body"
-    ],
+    BigIntLiteral: [],
     BinaryExpression: [
         "left",
         "right"
     ],
+    BlockStatement: [
+        "body"
+    ],
     BreakStatement: [
         "label"
     ],
-    CallExpression: [
-        "callee",
-        "arguments"
-    ],
     CatchClause: [
-        "param",
-        "body"
```

Comment:
The order is important here as it indicates which property should be traversed first. As such, we can change the order to alphabetical. (Same comment throughout this file.)

Reply:
Ok, sure, understood. But if the AST definers can't be convinced to define the properties in the order of iteration (taking into account nested interfaces, etc.), we'd have to add our own logic on how to sort these (and it might not be correct once new properties are added).